18/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM buildpack-deps:jessie
 
-ENV OTP_VERSION="18.3.4.2"
+ENV OTP_VERSION="18.3.4.3"
 
 # We'll install the build dependencies for erlang-odbc along with the erlang
 # build process:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-$OTP_VERSION.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="9c3443c378115c271d0ce214f404f4f602aeec196040ce4a4e8b2707ac0fd4b0" \
+	&& OTP_DOWNLOAD_SHA256="9cfb0315ae2650c3b1bb6f66f430753593f014698fd6105ef19cfd8eb90f72fe" \"
```

Comment:
please have a look the CI build failure log, looks like this line has a superfluous `"`

https://travis-ci.org/c0b/docker-erlang-otp/jobs/151225306#L227